pkg/ddevapp/ddevapp.go

```diff
@@ -2850,14 +2846,13 @@ func (app *DdevApp) Stop(removeData bool, createSnapshot bool) error {
 		}
 	}
 
-	if globalconfig.DdevGlobalConfig.IsTraefikRouter() && status == SiteRunning {
```

Comment:
`status == SiteRunning` should still be here.

Reply:
Thanks, good eye, fixed.